### PR TITLE
HPCC-16219 Result Monitoring - Also generate a direct comparison query

### PR DESCRIPTION
**Pull request by afishbeck**
Signed-off-by: Anthony Fishbeck anthony.fishbeck@lexisnexis.com

### CODE REVIEW

**Comment by richardkchapman** [comment]
@rpastrana Please review

File: esp/xslt/esdl2monitor.xslt [hunk 1]
```diff
@@ -21,6 +21,7 @@
     <xsl:param name="sourceFileName" select="'UNKNOWN'"/>
     <xsl:param name="responseType" select="''"/>
     <xsl:param name="requestType" select="''"/>
+    <xsl:param name="diffmode" select="'Monitor'"/>
```
Comment (rpastrana) [inline esp/xslt/esdl2monitor.xslt]: default behavior is Monitor if diffmode not set, but does the tool ever set diffmode to 'Monitor'?

**Review by rpastrana — APPROVED** [review]
@afishbeck looks reasonable, only one minor comment.

File: esp/xslt/esdl2monitor.xslt (hunk 1 above)
Comment (afishbeck) [inline esp/xslt/esdl2monitor.xslt]: @rpastrana The tool treats 'Monitor' as the true default.  It runs it without specifying once to generate the monitor, and runs it specifying 'Compare' to generate a compare query.

**Comment by afishbeck** [comment]
@richardkchapman This pull request can be merged.

**Pull request merged**